src/sentry/conf/server.py

```diff
@@ -1877,6 +1877,8 @@ def custom_parameter_sort(parameter: dict) -> tuple[str, int]:
     "organizations:standalone-span-ingestion": False,
     # A single flag for all the new performance UI that relies on span ingestion
     "organizations:spans-first-ui": False,
+    # Hide transactions and display spans
```

Comment:
In checkout flows, or _everywhere_? If it's the former, could you make the flag more specific? Otherwise it'll make a lot of upcoming Performance flags more confusing

Reply:
this is currently going to be used on the org stats page, checkout flows have access to the subscription data categories. but i could see it being used anywhere that we need to know if the org has access to spans (outside of billing)

Reply:
maybe spans-usage-tracking